src/cloud-api-adaptor/test/e2e/README.md

```diff
@@ -48,6 +48,27 @@ To leave the cluster untouched by the execution finish you should export `TEST_T
 
 To use existing cluster which have already installed Cloud API Adaptor, you should export `TEST_INSTALL_CAA=no`.
 
+## Attestation and KBS specific
+
+To deploy the KBS service and test attestation related cases, set `DEPLOY_KBS=yes`.
+To use a customized OPA policy file in KBS service, specify the OPA file via `TEE_CUSTOMIZED_OPA=/path/policy.rego`.
+For example, we need set `TEE_CUSTOMIZED_OPA=/$trustee_path/kbs/sample_policies/allow_all.rego` when test sample TEE as default policy disabled sample TEE.
+
+We also need artificats from trustee when do attestation test, to prepare trustee, do as following. 
+
+```
+pushd ${cloud-api-adaptor}/test/e2e
+git clone https://github.com/confidential-containers/trustee.git
+pushd trustee
+git checkout $COMMIT
+pushd kbs
+make CLI_FEATURES=sample_only cli
+popd
+popd
+popd
+```
+Note: we're using $COMMIT 65ee7e1acccd13dcb515058e71c5f8bfb4281e35 when writing this guide, it might change.
```

Comment:
Can we update the documentation to read the version from https://github.com/confidential-containers/cloud-api-adaptor/blob/16768ab6ba2606b6d504d2f7e10c70cbccf553b6/src/cloud-api-adaptor/versions.yaml#L46 please as that is supposed to be the stable and tested version.

Reply:
FYI: I ran `cloud-api-adaptor/src/cloud-api-adaptor/test/e2e/trustee# git checkout $(../../../hack/yq-shim.sh '.git.kbs.reference' ../../../versions.yaml)` instead of line 63, which is messy, but seems to work

Reply:
It's better, fixed, thanks! @stevenhorsman